Dockerfile

```diff
@@ -40,7 +40,7 @@ COPY . /code/
 RUN --mount=type=cache,target=/root/.npm npm install
 RUN npm run build
 
-RUN python manage.py collectstatic --noinput
+RUN python manage.py collectstatic --noinput --clear
```

Comment:
This is minor but fixes a warning when building the container locally. Since the static directory won't exist when GitHub Actions is building it for deployment, this only helps dev.